application/F3DStarter.cxx

```diff
@@ -249,10 +249,10 @@ class F3DStarter::F3DInternals
         return ss.str();
       };
       cameraMetadata << "{\n";
-      cameraMetadata << "  \"pos\": " << vec3toJson(state.pos) << ",\n";
-      cameraMetadata << "  \"foc\": " << vec3toJson(state.foc) << ",\n";
-      cameraMetadata << "  \"up\": " << vec3toJson(state.up) << ",\n";
-      cameraMetadata << "  \"angle\": " << state.angle << "\n";
+      cameraMetadata << "  \"pos\": " << vec3toJson(state.position) << ",\n";
+      cameraMetadata << "  \"foc\": " << vec3toJson(state.focalPoint) << ",\n";
+      cameraMetadata << "  \"up\": " << vec3toJson(state.viewUp) << ",\n";
+      cameraMetadata << "  \"angle\": " << state.viewAngle << "\n";
```

Comment:
```suggestion
      cameraMetadata << "  \"pos\": " << vec3toJson(state.position) << ",\n";
      cameraMetadata << "  \"foc\": " << vec3toJson(state.focalPoint) << ",\n";
      cameraMetadata << "  \"up\": " << vec3toJson(state.viewUp) << ",\n";
      cameraMetadata << "  \"angle\": " << state.viewAngle << "\n";
```
should the metadata keys change too? and if so should they be `camleCase` or `snake_case`?

Reply:
Lets use the same as the C++ API